packages/cashier/src/types/shared/account.types.ts

```diff
@@ -1,23 +1,45 @@
 /* -------------------------------------------------------------------------- */
 /* ACCOUNT TYPES                                                              */
 /* -------------------------------------------------------------------------- */
-import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+import { DetailsOfEachMT5Loginid, TransferBetweenAccountsResponse } from '@deriv/api-types';
 
 export type TAccount = {
     balance?: string | number;
     currency?: string;
     disabled?: boolean;
+    error?: JSX.Element | string;
+    is_crypto?: boolean;
     is_dxtrade?: boolean;
     is_mt?: boolean;
     market_type?: string;
-    nativepicker_text: string;
+    nativepicker_text?: string;
     platform_icon?: string;
-    text: JSX.Element | string;
-    value?: string;
+    text?: JSX.Element | string;
+    value?: string | undefined;
 };
```

Comment:
can we somehow avoid of using 'undefined' here?
maybe it is better to set initial values for this object? let say to just ''
better to check if this value is not 'undefined' before using it.
how do you think?

Reply:
Removed.